tests/core/test_config.py

```diff
@@ -731,104 +729,83 @@ def test_Config_export_config(caplog, shared_datadir, auto, expected_log_entries
-                        "data": {
-                            "variable": [
-                                {
-                                    "file": str(_ABS_PATH / "file.txt"),
-                                    "var_name": "my_path",
-                                },
-                            ]
-                        }
-                    },
-                },
-                times=2,
-            ),
-            id="leave_abs_paths_unchanged",
-        ),
-        pytest.param(
-            *repeat(
-                {
-                    "core": {
-                        "data": {
-                            # NB: Missing "file" key
-                            "variable": [
-                                {
-                                    "var_name": "my_path",
-                                },
-                            ]
-                        }
-                    },
-                },
-                times=2,
-            ),
-            id="ignore_missing_file_key",
-        ),
-        pytest.param(
-            *repeat(
-                {
-                    "core": {
-                        "data": {
-                            # NB: Not a list, as it should be
-                            "variable": {
-                                "file": "file.txt",
-                                "var_name": "my_path",
-                            },
-                        }
-                    },
-                },
-                times=2,
-            ),
-            id="variable_not_list",
-        ),
+            ["file.txt", "file2.txt"],
+            ["path/to/config/file.txt", "path/to/config/file2.txt"],
+            id="co-located",
+        ),
+        pytest.param(
+            ["data/file.txt", "data/file2.txt"],
+            ["path/to/config/data/file.txt", "path/to/config/data/file2.txt"],
+            id="inside_cfg_dir",
+        ),
+        pytest.param(
+            ["../data/file.txt", "../data/file2.txt"],
+            ["path/to/data/file.txt", "path/to/data/file2.txt"],
+            id="outside_cfg_dir",
+        ),
+        pytest.param(
+            ["../../../data/file.txt", "../../../data/file2.txt"],
+            ["data/file.txt", "data/file2.txt"],
+            id="moar_outside_cfg_dir",
+        ),
+        # pytest.param(
+        #     False,
+        #     [{"file": str(_ABS_PATH / "file.txt"), "var_name": "my_path"}],
+        #     [{"file": str(_ABS_PATH / "file.txt"), "var_name": "my_path"}],
+        #     id="leave_abs_paths_unchanged",
+        # ),
+        # pytest.param(
+        #     False,
+        #     [{"var_name": "my_path"}],
+        #     [{"var_name": "my_path"}],
+        #     id="ignore_missing_file_key",
+        # ),
+        # pytest.param(
+        #     False,
+        #     {"file": "file.txt", "var_name": "my_path"},
+        #     {"file": "file.txt", "var_name": "my_path"},
+        #     id="variable_not_list",
+        # ),
```

Comment:
I guess either comment these back in or delete them

Reply:
This reminds me of the problem we had with multiple config files (one from each model) has this been resolved?

Reply:
Nope that issue (#234) hasn't been resolved, multiple config files are completely, but all of the instructions of where to load data from have to be contained in a single config file